lib/logstash/bundler.rb

```diff
@@ -0,0 +1,125 @@
+require "bundler"
```

Comment:
+1 on New directory, and we can move the socket patch there and maybe have
`logstash/patches/windows/XXX.rb`